cmd/plume/gce.go

```diff
@@ -0,0 +1,260 @@
+// Copyright 2015 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
```

Comment:
This is the bit I was referring to about having two license headers. Just need something like:

```
// Copyright 2015 CoreOS, Inc.
// Copyright 2015 The Go Authors.
//
// Licensed under the Apache License.....
```